NuGetReleaseTool/NuGetReleaseTool/GenerateReleaseNotesCommand/README.md

```diff
@@ -0,0 +1,16 @@
+# generate-release-notes
```

Comment:
Should this file be moved up a directory to the solution, rather than the project? It might make it a little more visible?

Reply:
Currently, I have a README in each command directory, then in the solution directory I have a mega readme that links to the readmes for each command.

Reply:
Basically, the total readme count should be 4.